src/libcommon/comm.h

```diff
@@ -123,6 +124,172 @@ enum class RequestNum {
     RECONSIDER_SKIPPED_UPDATE,
     UTILITY_QUIT,
 };
+inline std::string enumClassToString(RequestNum e) {
```

Comment:
Why not putting the implementation in `types.cpp`?

Reply:
The idea was to have only the enum from types.h in types.cpp. In fact, it is possible to add the converters for RequestNum and SignalNum directly in types.h as it is the same lib, but it may be better to keep them close to the enum they are referencing. For example, if an enum is defined in the libcommon server, we would not be able to set the converter in types.cpp as is in libcommon and does not import libcommonserver.